pkg/app/carbonapi/http_handlers.go

```diff
@@ -944,39 +945,139 @@ func (app *App) findHandler(w http.ResponseWriter, r *http.Request, lg *zap.Logg
 		logLevel = zapcore.ErrorLevel
 		return
 	}
-
+	// TODO: extend writeResponse w/new formats and migrate to it
```

Comment:
Not needed in this change, I just found it very confusing that we have jsonp wrapper implemented in `writeResponse` already and find handler using separate wrapper for some reason.